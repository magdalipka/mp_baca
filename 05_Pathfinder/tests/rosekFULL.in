5 5
1 1 1 0 0
0 0 0 0 1
1 1 0 1 1
0 1 0 0 1
0 0 0 1 0
12
r 0 3 0 1
r 0 0 4 4
r 4 0 4 4
r 3 1 4 4
r 4 0 4 0
r 2 0 3 3
i 0 3 0 1
i 0 0 4 4
i 4 0 4 4
i 3 1 4 4
i 4 0 4 0
i 2 0 3 3

