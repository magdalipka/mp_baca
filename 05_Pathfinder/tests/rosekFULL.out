r E E S S S W W N
r E E N N N E N E
r X
r W N N E N E
r
r N N N E
i E E S S S W W N
i E E N N N E N E
i X
i W N N E N E
i
i N N N E
